package-lock.json

```diff
@@ -1,4636 +1,8 @@
 {
   "name": "r",
   "version": "1.6.6",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
The change in lock file seems to generate much bigger diff than it really has. Anything we could do about it? Somebody else runs `npm install` next time and the lock file could probably change back and forth.

Reply:
Not really sure what's going on here. I think I merged a different lockfile from master at some point or accidently changed this at some point. I can just revert this change for now.

Long term it might make sense to switch to yarn anyways.